PYME/misc/shellutils.py

```diff
@@ -456,7 +456,7 @@ def savitzky_golay(y, window_size, order, deriv=0):
     try:
         window_size = np.abs(np.int(window_size))
         order = np.abs(np.int(order))
-    except ValueError, msg:
+    except ValueError:
```

Comment:
This is really odd - why are we catching a `ValueError` and then raising a `ValueError` ??

Reply:
Looks like to raise it with a custom message